GW2EIEvtcParser/ParsedData/Agents/AgentItem.cs

```diff
@@ -470,5 +470,29 @@ public bool IsSpecies(ArcDPSEnums.ChestID id)
         {
             return IsSpecies((int)id);
         }
+
+        public bool IsAnySpecies(IReadOnlyList<ArcDPSEnums.TrashID> ids)
+        {
+            foreach (ArcDPSEnums.TrashID id in ids)
```

Comment:
Can be simplified as "return ids.Any(x => IsSpecies(id));"
Same for the one below.